packages/ui/src/theme/css/component/AIConversation/aiConversation.scss

```diff
@@ -0,0 +1,144 @@
+.amplify-ai-conversation {
```

Comment:
Would prefer to see all style files for the ai conversation component kept in its package